lib/solitude/api.py

```diff
@@ -592,8 +592,8 @@ def create_transaction(self, generic_buyer, generic_seller,
         # In the case of Zippy, it does not detect any of these values
         # itself. All other providers will detect these values without
         # help from Webpay.
-        carrier = 'USA_TMOBILE'
-        region = '123'
+        carrier = 'FAKE'
+        region = 'CAN'
```

Comment:
We actually get back from bango a network and region like USA_TMOBILE. in solitude its just a charfield. As long as we get back something that conforms to ISO 3166-1 and can easily be run through mobile-codes, I'm happy :)

Also long term I don't know if mcc makes sense for non-SIM enabled devices.